actuators/machine.go

```diff
@@ -128,18 +122,18 @@ func (m *Machine) Create(ctx context.Context, c *clusterv1.Cluster, machine *clu
 	}
 
 	m.Log.Info("Creating a new worker node")
-	worker, err := actions.AddWorker(c.Name, machine.GetName(), machine.Spec.Versions.Kubelet)
+	worker, err := actions.AddWorker(c.Name, machine.GetName(), *machine.Spec.Version)
 	if err != nil {
 		m.Log.Error(err, "Error creating new worker node")
 		return err
 	}
 	providerID := actions.ProviderID(worker.Name())
 	machine.Spec.ProviderID = &providerID
-	return m.save(old, machine)
+	return nil
```

Comment:
why do we not need to save anymore?

Reply:
The actuators are going away and being replaced by the controllers. I could have deleted *all* of this code, but I wanted to keep the majority of it around to reuse in the controller if necessary.

The save went away because kubebuilder v0.2 has completely changed how they do patches for objects and needs to be rewritten.